app/controllers/admin/project_logs_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::ProjectLogsController < Admin::AdminController
+  def destroy
+    @log = ProjectLog.find(params[:id])
+    @step = ProjectStep.find(@log.project_step_id)
+    @loan = Loan.find(@step.project_id)
+
+    if @log.destroy
+      redirect_to admin_loan_path(@loan), notice: I18n.t(:notice_deleted)
```

Comment:
This takes us back to the default view for a loan page, which shows the details tab. However, when a project log is deleted, we are on the timeline tab under a specific project step. Not sure if we want this redirect to reflect a desired page flow. And if so, not sure how to go about doing that.